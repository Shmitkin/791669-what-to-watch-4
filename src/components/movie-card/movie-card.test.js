import React from "react";
import renderer from "react-test-renderer";
import MovieCard from "./movie-card.jsx";

it(`Should MovieCard render correctly`, () => {
  const tree = renderer
    .create(
        <MovieCard
          movieTitle={`Stranger Things`}
          moviePreview = {`img/stranger-thisngs.jpg`}
          onMovieCardHover = {() => {}}
          onMovieCardUnhover = {() => {}}
          onMovieCardClick = {() => {}}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
