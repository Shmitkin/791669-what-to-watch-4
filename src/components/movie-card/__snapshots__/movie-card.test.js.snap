// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieCard render correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height="175"
      loop={true}
      muted={true}
      poster="img/pulp-fiction.jpg"
      src="img/seven-years-in-tibet.mp4"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Pulp Fiction
    </a>
  </h3>
</article>
`;
